projects/optic/src/commands/capture/actions/documented.ts

```diff
@@ -190,6 +200,91 @@ function summarizePatch(
   return [];
 }
 
+function summarizeUnpatchableDiff(
+  diff: GroupedUnpatchableDiff,
+  parseResult: ParseResult,
+  options: {
+    mode: 'update' | 'verify';
+    verbose: boolean;
+  }
+): string[] {
+  const verbose = options.verbose && options.mode === 'verify';
+  const pointerLogger = jsonPointerLogger(parseResult.sourcemap);
+  const { path, validationError, examples } = diff;
+  const location = locationFromPath(path);
+  const summarizedExamples =
+    examples
+      .slice(0, 3)
+      .map((e) => JSON.stringify(e))
+      .join(', ') +
+    (examples.length > 3 ? ` and ${examples.length - 3} other values` : '');
+
+  const lines = [
+    chalk.red(
+      `${location} schema (${diff.schemaPath}) with keyword '${
+        validationError.keyword
+      }' and parameters ${JSON.stringify(
+        validationError.params
+      )} received invalid values ${summarizedExamples}`
+    ),
+  ];
+  if (options.mode === 'update') {
+    lines.push(chalk.red(` ⛔️ schema could not be automatically updated`));
```

Comment:
if a user tries to update something we cannot patch (could be because we haven't built support in yet, or it's not possible), we surface this error showing that it could not be automatically updated.